styles/styles.css

```diff
@@ -995,6 +1044,46 @@ main .section.padded .section-container {
     font-size: var(--heading-font-size-mxl);
   }
 
+  /* section - social */
+  main .section.social-media {
+    padding: 5rem 0;
+  }
+
+  main .section.social-media .section-container {
+    display: flex;
+  }
+
+  main .section.social-media .default-content-wrapper {
+    width: 100%;
+  }
+
+  main .section.social-media a::after {
+    content: "";
+    height: 0.625rem;
+    width: 0.375rem;
+    display: inline;
+    background-image: url("../../icons/arrow-down-gray.svg");
+    background-size: contain;
+    margin-left: 0.5rem;
+    margin-top: 0.25rem;
+    background-repeat: no-repeat;
+    transform: rotate(270deg);
```

Comment:
Should we maybe add a separate icon rather than rotating this one?

Reply:
Any benefit to avoid using rotating?

Reply:
Consistency? It's a small thing though, I agree